menoh/menoh.cpp

```diff
@@ -102,6 +102,92 @@ menoh_error_code menoh_make_model_data_from_onnx_data_on_memory(
         return menoh_error_code_success;
     });
 }
+menoh_error_code menoh_make_model_data(menoh_model_data_handle* dst_handle) {
+    return check_error([&]() {
+        *dst_handle = std::make_unique<menoh_model_data>().release();
```

Comment:
Is it common in C++ to just construct a value using `std::make_unique` and immediately call `release`?

Reply:
It is paranoiac code to avoid `new` keyword. I have no idea that it is common but we can avoid `new`.

Reply:
I see.